### PR TITLE
dynamic launchd and profiles prefix with node['organization']

### PR DESCRIPTION
**Pull request by discentem**

### CODE REVIEW

**Comment by erikng** [comment]
This is technically one more line of code than before but I get it. :P

File: chef/cookbooks/cpe_init/recipes/company_init.rb [hunk 1]
```diff
@@ -15,9 +15,10 @@
 # HERE: This is where you would set attributes that are consumed by the API
 # cookbooks.
 # Be sure to replace all instances of MYCOMPANY with your actual company name
-node.default['organization'] = 'MYCOMPANY'
-node.default['cpe_launchd']['prefix'] = 'com.MYCOMPANY.chef'
-node.default['cpe_profiles']['prefix'] = 'com.MYCOMPANY.chef'
+node.default['organization'] = 'YOURCOMPANY'
```
Comment (natewalck) [inline chef/cookbooks/cpe_init/recipes/company_init.rb]: Keep it MYCOMPANY or change the comment to match the new value.

**Comment by discentem** [comment]
@natewalck It tested it, seems to work fine. The munki profile that's installed is named correctly based on the prefix in my revisions.

**Comment by natewalck** [comment]
Woot. LGTM!

**Pull request merged**